doc/man/8/rbd.rst

```diff
@@ -946,6 +959,14 @@ To unmap an image::
 
        rbd device unmap /dev/rbd0
 
+To attach an image via the nbd device type::
+
+       rbd device attach mypool/myimage --device /dev/nbd0 --device-type nbd
```

Comment:
@idryomov I had intentionally not captured `--force` option in the above example, someone who's lazy might just copy-paste the command and not notice why there is a force option.